test/List.js

```diff
@@ -76,7 +76,7 @@ List.prototype[FL.reduce] = function(f, x) {
 List.prototype[FL.traverse] = function(f, of) {
   return this.tag === 'Nil' ?
     of(Nil) :
-    Z.ap(Z.map(curry2(Cons), f(this.head)), Z.traverse(of, f, this.tail));
+    Z.ap(Z.map(curry2(Cons), f(this.head)), this.tail[FL.traverse](f, of));
```

Comment:
`of` is not necessarily `List['fantasy-land/of']` (in fact, it almost certainly is not). We have no way of getting from `of` to a type representative, so we're forced to use the `of`-based API rather than our API based on type representatives.

If we update [`fantasy-land/traverse`][1] to take a type representative this problem will disappear. ;)


[1]: https://github.com/fantasyland/fantasy-land#traverse-method